src/json.h

```diff
@@ -18,6 +18,33 @@
 #include <vector>            // std::vector
 #include <iterator>          // std::iterator
 
+#define JSON_NO_RETURN __attribute__((noreturn))
+#define JSON_USE_LITERALS
+
+#if defined(_MSC_VER)
+
+#include <cstdint>
+
+#if _MSC_VER < 1900
+#define noexcept throw()
+#define u8
+#undef JSON_USE_LITERALS
+#endif
+
+#undef JSON_NO_RETURN
+#define JSON_NO_RETURN __declspec(noreturn)
+
+#define or ||
+#define and &&
+#define not !
```

Comment:
Why not `#include <ciso464>` which contains all defines?

http://www.cplusplus.com/reference/ciso646/